pretixprint/app/src/main/res/layout/fragment_choose_byte_protocol.xml

```diff
@@ -32,27 +31,11 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/textView5" />
 
-    <Button
-        android:id="@+id/btnNext"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="16dp"
-        android:layout_marginBottom="16dp"
-        android:text="@string/btn_next"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
-
-    <Button
-        android:id="@+id/btnPrev"
-        android:layout_width="wrap_content"
+    <include
```

Comment:
This layout no longer works correctly (shows no choices). previously, as the recyclerview has

app:layout_constraintBottom_toTopOf="@+id/btnNext"

and the include has

app:layout_constraintTop_toBottomOf="@id/list"

Not sure if this is related to the include or due to the constraints themselves

Reply:
Yep, somehow accessing btnNext didn't work in all cases. I just gave all the includes their own id, so they can be referenced by `app:layout_constraintBottom_toTopOf` - and it seems to work even on very very small devices

<img width="239" alt="Screenshot 2023-08-10 at 10 15 11" src="https://github.com/pretix/pretixprint-android/assets/172415/011e5999-ae6f-4af8-9005-020cef79b5f7">